apps/notifications/index.jsx

```diff
@@ -279,10 +274,11 @@ export class Notifications extends Component {
 export default connect(
 	( state ) => ( {
 		currentLocaleSlug: getCurrentLocaleVariant( state ) || getCurrentLocaleSlug( state ),
+		forceRefresh: shouldForceRefresh( state ),
```

Comment:
@sgomes I don't really know what I'm doing here and would love your help/advice. 😆 This currently doesn't work. 😞 

_Context_: I'm trying to avoid having to use a window event to tell the Notifications panel component to refresh itself. See the implementation in this file and in `client/state/notification-panel` ([link](https://github.com/Automattic/wp-calypso/pull/46070/files#diff-785bc12dc015913ac3fbddfa88d413abR7)). Is using redux here possible, or is a window event the only option?